docs/docs/migrating-from-v0-to-v1.md

```diff
@@ -24,24 +24,24 @@ git mv utils src
 
 ## Replace react-router's Link component with gatsby-link
 
-Add new package to project.
+`gatsby-link` is a wrapper for the `<Link>` component in react-router.  It automatically prefixes urls and handles prefetching on browsers that don't support ServiceWorker.  Gatsby doesn't include it, so you should install it:
```

Comment:
A nit, it prefetches regardless of whether you're using a sw or not.

Also change the last sentence to something like:  "Add `gatsby-link` to your project by running:"